packages/authgear-core/src/container.ts

```diff
@@ -89,6 +92,16 @@ export abstract class BaseContainer<T extends BaseAPIClient> {
    */
   expireAt?: Date;
 
+  /**
+   * @internal
+   */
+  sessionState: SessionState;
```

Comment:
Both the session state value and the listeners are suppsoed to be used by developers, so we should make them public.